questionGenerator.py

```diff
@@ -0,0 +1,291 @@
+import pandas as pd
+from utils import dataLoader, dataSaver, printAndLog
+from config import OpenAIBot
+import json
+import sys
+
+class QuestionData:
+    """
+    Represents a class that handles question data generation.
+
+    Args:
+        config (Config): The configuration object.
+        videoData (VideoData): The video data object.
+        topicModeller (TopicModeller): The topic modeller object.
+        load (bool, optional): Whether to load existing question data. Defaults to True.
+    """
+
+    def __init__(self, config, videoData, topicModeller, load=True):
+        self.config = config
+
+        self.clusteredTopics = getClusteredTopics(videoData, topicModeller)
+        self.dominantTopics = getDominantTopic(self.clusteredTopics)
+        self.relevantText, self.questionText = getTextAndQuestion(
+            self.dominantTopics, videoData
```

Comment:
Log the question and text data from the output.

Reply:
Addressed in the [new Commit](https://github.com/tl-its-umich-edu/annoto-gai/pull/16/commits/be278feff6e0c397bf70603df9d69149c0fcf026).